packages/syft/tests/syft/types/dicttuple_test.py

```diff
@@ -39,9 +39,7 @@ def test_dict_tuple_not_subclassing_mapping():
 
 @pytest.mark.parametrize("dict_tuple", SIMPLE_TEST_CASES)
 def test_should_iter_over_value(dict_tuple: DictTuple) -> None:
-    values = []
-    for v in dict_tuple:
-        values.append(v)
+    values = list(dict_tuple)
```

Comment:
I think here we're testing this specific syntax.

Reply:
thx

Reply:
Looks like this is still not reverted. Is it auto-formatted by a linter rule? If so you can use a `# noqa: XXX` with `XXX` being the specific linter rule to prevent it from auto-formatting.